python/lsst/ip/diffim/modelPsfMatch.py

```diff
@@ -416,8 +416,7 @@ def _buildCellSet(self, exposure, referencePsfModel):
 
         sciencePsfModel = exposure.getPsf()
 
-        dimenR = referencePsfModel.getLocalKernel().getDimensions()
-        psfWidth, psfHeight = dimenR
+        dimenR = referencePsfModel.getLocalKernel(scienceBBox.getCenter()).getDimensions()
```

Comment:
This is probably fine. In other places we have used `sciencePsfModel.getAveragePosition()` instead of `scienceBBox.getCenter()`, but I think it makes sense to use the center of the science image.

Reply:
I have changed it to use the center of the bbox explicitly. This makes a significant difference, say, when one passes in a postage stamp of an exposure. The PSF's average position may be far away from the stamp that is provided.